package-lock.json

```diff
@@ -115,6 +115,7 @@
     "apps/care_livekit_fe": {
       "name": "care-livekit",
       "version": "0.0.1",
+      "extraneous": true,
```

Comment:
why do we have changes in `package-lock.json` 🤔

Reply:
No idea, I didn't add it manually. I think that got updated after running `` npm run install-all`` 🤔

Reply:
You can discard the package-lock.json changes if no packages has been added/removed/updated before making commits.